packages/shared-components/src/assets/i18n/en.json

```diff
@@ -42,5 +42,27 @@
   "menu.guides.label": "Interactive guides",
   "menu.plugins.label": "Plugins",
   "visual.graph.label": "Visual graph",
-  "view.rdf.rank.title": "RDF Rank"
+  "view.rdf.rank.title": "RDF Rank",
+  "language_selector": {
+    "toggle_menu": {
+      "label": "Select language",
+      "tooltip": "Current language: English"
```

Comment:
uhmm what are these?!

Reply:
This will be achieved through [GDB-11275](https://ontotext.atlassian.net/browse/GDB-11275).